Lib/fontbakery/profiles/googlefonts.py

```diff
@@ -5941,6 +5942,45 @@ def com_google_fonts_check_metadata_can_render_samples(ttFont, family_metadata):
        yield PASS, "OK."
 
 
+@check(
+    id = "com.google.fonts/check/metadata/category_hints",
+    rationale = """
+        Sometimes the font familyname contains words that hint at which is the most likely correct category to be declared on METADATA.pb
+    """,
+    conditions = ["family_metadata"],
+    proposal = 'https://github.com/googlefonts/fontbakery/issues/3624'
+)
+def com_google_fonts_check_metadata_category_hint(family_metadata):
+    """Check if category on METADATA.pb matches what can be inferred from the family name."""
+
+    HINTS = {
+        "SANS_SERIF": ["Grotesk",
+                       "Grotesque"],
+        "SERIF": ["Old Style",
+                  "Transitional",
+                  "Garamond"],
+        "DISPLAY": ["Display"],
+        "HANDWRITING": ["Hand",
```

Comment:
"Handwriting" should be in this list :)

Reply:
"Handwriting" will already be detected by the "Hand" hint keyword.